packages/cli/.env.example

```diff
@@ -0,0 +1,4 @@
+GITHUB_PAT=GITHUB_PAT
```

Comment:
can we avoid hardcoded design dependency on Github?
considering that we have most of our repos moving to Harness Code now, we should support querying harness0.harness.io with the Harness PAT as well.

a simpler, more generic design would be to just take a complete url as a param, instead of taking org + repo + branch and constructing github urls via "magic".